js/app/widgets/eknWebview.js

```diff
@@ -176,9 +177,31 @@ const EknWebview = new Knowledge.Class({
                 if (scheme === 'license') {
                     let license = GLib.uri_unescape_string(uri.replace('license://', ''), null);
                     uri = Endless.get_license_file(license).get_uri();
+
+                    let license_view = new EosKnowledgePrivate.WebViewDialog({
+                      'transient-for' : this.get_toplevel(),
```

Comment:
Please remove quotes and use underscores for these parameters (e.g. `transient_for`)